### ISSUE TITLE
Atom had replaced my custom theme with Atom Dark

### ISSUE THREAD

**Issue by GingerN** [atom/atom#15064]
I'm using Atom for many monthes. And I've created my custom theme. Unfortunatly I've created it only on my local machine. I've choosen it some time ago and today I decided to try some other theme changing it in settings. When I've changed to another theme - it has changed, but when I wanted to change back to my custom theme, I found out that my custom theme was replaced with Atom Dark. I've checked files in c/users/username/github folder and they realy were replaced with Atom Dark. I have not expected this behaviour at all. Is there a way to restore files?

**Comment by lee-dohm** [comment]
It sounds like rather than placing your custom theme in the `%USERPROFILE%\.atom\packages` directory, you edited the files that were installed with Atom itself? If so, when Atom updated to a new version it very well could have overwritten the changes you made. Additionally, if my assumption is correct, then I don't know of a way to recover your customizations.

For future reference, you may want to take a look at the [Atom Flight Manual](http://flight-manual.atom.io) and its [tutorial on how to create a theme](http://flight-manual.atom.io/hacking-atom/sections/creating-a-theme/). Following the instructions there will show you how to create a custom theme in such a way that it won't be overwritten when Atom updates.

**Comment by GingerN** [comment]
I have created it via exactly that manual the link you've posted. The theme was created in c/users/username/github/theme-name folder.

**Comment by lee-dohm** [comment]
There's no code in Atom that I'm aware of that would randomly replace the contents of a theme directory with the contents of another theme. If you can reproduce the problem and give us some indication of how this happened, I'll be happy to take a closer look.

**Comment by GingerN** [comment]
Atom version was 1.17.2. I have changed syntax theme from file->settings->themes - syntax theme. I've tried different others and Atom Dark too. After I've changed back to my theme (called "My"), it changed back to Atom Dark. After several attempts, it stayed at "My" but with Atom Dark syntax. I've turned off, turned on Atom and tried to set "My" syntax theme again - theme is choosen but synax is Atom Dark's. I've checked files in c/users/username/github/mythemename folder and they realy were replaced with Atom Dark. I've created "My" theme with this tutorial http://flight-manual.atom.io/hacking-atom/sections/creating-a-theme/.
